{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Source \"@openzeppelin/contracts/access/Ownable.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c3/c3e196186b09d7348c0bf66be6428c2a7420fa74_MiningV2.sol:9:1:\n  |\n9 | import \"@openzeppelin/contracts/access/Ownable.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/c3/uniswapv2/interfaces/IUniswapV2Pair.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c3/c3e196186b09d7348c0bf66be6428c2a7420fa74_MiningV2.sol:34:1:\n   |\n34 | import './uniswapv2/interfaces/IUniswapV2Pair.sol';\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/c3/IRewardDistributionRecipient.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c3/c3e196186b09d7348c0bf66be6428c2a7420fa74_MiningV2.sol:35:1:\n   |\n35 | import './IRewardDistributionRecipient.sol';\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/c3/IMintable.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c3/c3e196186b09d7348c0bf66be6428c2a7420fa74_MiningV2.sol:36:1:\n   |\n36 | import './IMintable.sol';\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/access/Ownable.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c3/c3e196186b09d7348c0bf66be6428c2a7420fa74_MiningV2.sol:37:1:\n   |\n37 | import \"@openzeppelin/contracts/access/Ownable.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/math/SafeMath.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c3/c3e196186b09d7348c0bf66be6428c2a7420fa74_MiningV2.sol:38:1:\n   |\n38 | import \"@openzeppelin/contracts/math/SafeMath.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/GSN/Context.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/c3/c3e196186b09d7348c0bf66be6428c2a7420fa74_MiningV2.sol:212:1:\n    |\n212 | import \"../GSN/Context.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
